docs/container/README.md

```diff
@@ -77,6 +77,33 @@ Once the FalconContainer resource is pushed to the cluster the operator will sta
 | conditions.["InstallerComplete"]    | Informs about completion of Falcon Container Installer. Users can review the installer Job/Pod in `falcon-system-configure` namespace     |
 | conditions.["Complete"]             | Informs about the completion of the deployment of Falcon Container                                                                        |
 
+### Image Registry considerations
+
+Falcon Container Image is distributed by CrowdStrike through CrowdStrike Falcon registry. Operator supports two modes of deployment:
+
+#### (Option 1) Use CrowdStrike registry directly
+
+This is recommended for demoing purposes. Does not require any advanced set-ups. Users are advised to use the following excerpt in theirs FalconContainer custom resource definition.
+
+```
+registry:
+  type: crowdstrike
+```
+
+Falcon Container product will then be installed directly from CrowdStrike registry. Any new deployment to the cluster may contact CrowdStrike registry for the image download. The `imagePullSecret` needs to be added to every namespace to enable direct pull from CrowdStrike registry. By default the `imagePullSecret` will be created in `falcon-system-configure`, `falcon-system`, and `default` namespaces.
+
+#### (Option 2) Let operator mirror Falcon Container image to your local registry
+
+This is recommended for production. Requires advanced set-up to grant the operator push access to your local registry. The operator will then mirror Falcon Container image from CrowdStrike registry to your local registry of choice.
+
```

Comment:
Why is this only recommended for production and not demoing?

Reply:
Because the other one is easier for demoing. 🤣

Reply:
True. But let's just delete the sentence unless we need it?